lesson5/public/index.html

```diff
@@ -1,23 +0,0 @@
-//index file for xml
-<html lang = "en">
-  <head>
-    <meta charset = "UTF-8">
-    <title></title>
-  </head>
-  <body>
-      Name: <input type = 'text' id = 'nameBox'/>
-      Comment: <input type = 'text' id = 'commentBox'/>
-      <input type = 'submit' id = 'submitBox'>
-    <script>
-        const button = document.getElementById('submitBox');
-        button.onclick =()=>{
-          const name = document.getElementById('nameBox').value;
```

Comment:
is it indented, check it once again

Reply:
this file is part of lesson5 so I removed it. Please review the files under lesson6.